### PR TITLE
ActionAndFuncInCsharp solution added.

### PR DESCRIPTION
**Pull request by heisaduvan**
Unit tests added.

### CODE REVIEW

File: csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/Program.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ActionAndFunc
+{
+    public class Program
+    {
+        private const string Quote = "I am the one who knocks.";
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/Program.cs:8]: Please introduce some line breaks between fields, methods, and logical units in the methods. The code feels very stuffed.

File: csharp-advanced-topics/ActionAndFuncInCsharp/Tests/ActionandFuncUnitTests.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ActionAndFunc;
+namespace Tests
+{
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/ActionAndFuncInCsharp/Tests/ActionandFuncUnitTests.cs:4]: Same thing for line breaks.

File: csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/ActionAndFunc.csproj [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
```
Comment (CodeMazeBlog) [inline csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/ActionAndFunc.csproj:6]: This is okay for the evaluation, but we need to use .NET 6 for the production articles.

File: csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/Program.cs (hunk 1 above)
Comment (heisaduvan) [reply csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/Program.cs:8]: Sorry, i couldn't understand. Do you want this ? 
```
public class Program {

        private const string Quote = "I am the one who knocks.";
```

Comment (CodeMazeBlog) [reply csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/Program.cs:8]: Hey Isa, you should introduce newlines here:
image
And also here:
image
Applies to the test class as well.

**Pull request closed without merging**